### PR TITLE
Add relu int

### PR DESCRIPTION
**Pull request by mateuszchudyk**

### CODE REVIEW

File: postprocess.h [hunk 1]
```diff
@@ -5,6 +5,10 @@
 #include "types.h"
 #include "vec_utils.h"
 
+// TODO: We support some postprocess in few variations e.g. we support ReLU for
```
Comment (kpu) [inline postprocess.h]: Yes some notion of type would be useful.

File: intrinsics.h [hunk 2]
```diff
@@ -66,6 +66,12 @@ INTGEMM_SSE2 static inline __m128i madd_epi16(__m128i first, __m128i second) {
 INTGEMM_SSSE3 static inline __m128i maddubs_epi16(__m128i first, __m128i second) {
   return _mm_maddubs_epi16(first, second);
 }
+/*
```
Comment (kpu) [inline intrinsics.h]: Ugh I guess we could separate on SSE4.1 but agree it's not worth it for now.

File: postprocess.h [hunk 3]
```diff
@@ -203,6 +207,116 @@ class PostprocessImpl<ReLU, CPUType::AVX512BW> {
...
+ */
+class ReLU_int8 {};
+
+template <>
+class PostprocessImpl<ReLU_int8, CPUType::SSE2> {
+public:
+  using InputRegister = RegisterPair128i;
+  using OutputRegister = RegisterPair128i;
+
+  PostprocessImpl(const ReLU_int8& config) {}
```
Comment (kpu) [inline postprocess.h]: explicit

File: postprocess.h [hunk 4]
```diff
@@ -203,6 +207,116 @@ class PostprocessImpl<ReLU, CPUType::AVX512BW> {
...
+
+  INTGEMM_SSE2 inline OutputRegister run(InputRegister input, Index offset) {
+    static const auto const_zero = setzero_si<__m128i>();
+    return {
+      _mm_and_si128(_mm_cmplt_epi8(const_zero, input.pack0123), input.pack0123),
```
Comment (kpu) [inline postprocess.h]: Given that we're going to be repacking 32 bits to 8 bits, we may as well have this run on one register rather than two.

Comment (mateuszchudyk) [reply postprocess.h]: I changed it.

**Pull request merged**